docs/CombineWithPyPDF2.md

```diff
@@ -0,0 +1,102 @@
+# Combine with PyPDF2
+
+`fpdf2` cannot **parse** existing PDF files.
+
+However, other Python libraries can be combined with `fpdf2`
+in order to add new content to existing PDF files.
+
+This page provides several examples of doing so using [`PyPDF2`](https://github.com/py-pdf/PyPDF2).
+
+## Modifying content in an existing PDF
+
+```python
+import sys
```

Comment:
On which PDF file have you tested this script?
I have errors like this with all the PDF files I have tested:
```
  File "/home/user/.local/share/virtualenvs/fpdf2/lib/python3.8/site-packages/PyPDF2/generic/_data_structures.py", line 862, in read_object
PyPDF2.errors.PdfReadError: Invalid Elementary Object starting with b'P' @184
```

Reply:
I just created a test pdf file with fpdf2 before running the script, I also tested it on a sample pdf I grabbed off the internet. Attaching both the original and modified files below for your reference.  
[tuto1.pdf](https://github.com/PyFPDF/fpdf2/files/9752076/tuto1.pdf)
[sample.pdf](https://github.com/PyFPDF/fpdf2/files/9752048/sample.pdf)

Reply:
OK, my bad, the issue was due to a bug I introduced in my branch 😅

Reply:
ahh well, guess that solves it... I'll be making the changes you mentioned above in a few minutes. Excited for my first contribution to open source!